### ISSUE TITLE
Coordinate with @NikosAlexandris ?

### ISSUE THREAD

**Issue by ninsbl** [veroandreo/i.landsat#13]
Stumbled over this one just now:
https://github.com/NikosAlexandris/i.landsat.import
Maybe you can coordinate with @NikosAlexandris?

**Comment by veroandreo** [comment]
Yup, I have seen Nikos module and studied it a bit! It looks so much more complex than I thought for mine, which is more on the lines of a simplified and adapted version of i.sentinel.import. 

@NikosAlexandris, all your feedback is more than welcome! And thanks@ninsbl for the heads-up :)

**Comment by NikosAlexandris** [comment]
Ciao guys. Nice you work on this Vero.
If I can share anything that can be useful, that would be:
- Using classes is cool. But writing tests for classes will be harder than if you follow a functional approach.
- Testing, testing, testing (...I write this first for me :-)).
- https://github.com/NikosAlexandris/i.landsat.import/blob/master/identifiers.py and https://github.com/NikosAlexandris/i.landsat.import/blob/master/identify.py ideas/material you could use/recycle and save time?
- Since you are in the beginning of it, try to split-up the code and don't follow a monolithic approach. Smaller files, "thematically" organised, imported when needed. Debugging will be easier.
- Ideally, the CLI and the main code should be in separate files too.
- The main code should read as you read the pseudo-code of an algorith. What does the code do? And not how the code does what it does.

I had a quick look at my own script, and there is a lot of work for me there to do, so as to follow on the above points. But, OK :-).

Please feel free to recycle anything if and as you see fit. And if I can be useful in anything specific, I am here.
